src/Orleans.Runtime/GrainDirectory/CachedGrainLocator.cs

```diff
@@ -1,15 +1,14 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Configuration;
```

Comment:
This class has lots of conflicts, probably should manually merge

Reply:
yes, it's better to re-do the logic. 

the core logic I added is: check if it's global grain, if it's, use global membership talbe, if it's not, use the local member ship table.